src/app/cards/munchkin/cards/treasures/treasure-group.ts

```diff
@@ -0,0 +1,7 @@
+import { CardGroup } from 'src/app/cards/infrastructure/cards';
+
+export const Treasures: CardGroup = {
+  name: 'treasure',
```

Comment:
Should have `namePlural`?

Reply:
No need in it now and not sure if it will be used at some point, so leaving as is for now